raft/server.hh

```diff
@@ -23,17 +23,19 @@ public:
         // automatically snapshot state machine after applying
         // this number of entries
         size_t snapshot_threshold = 1024;
-        // how many entries to leave in the log after tacking a snapshot
+        // how many entries to leave in the log after taking a snapshot
         size_t snapshot_trailing = 200;
+        // limit on the total number of bytes, consumed by snapshot trailing entries,
+        // must be smaller than max_log_size
+        size_t max_snapshot_trailing_bytes = 1 * 1024 * 1024;
```

Comment:
We don't use _bytes for e.g. max_log_size. Please either use _bytes consistently or don't use it at all.

Reply:
snapshot_trailing_size